Added Flechette Shells

Masterdiekillsu opened the pull request:
High powered Shotgun only Ammo has been added!

Masterdiekillsu committed:
Added Flechette Shells

High powered Shotgun only Ammo has been added!

bleehu commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -601,6 +601,23 @@ biologic targets and armor.
 
+== Flechette Shells ($2.5 x mag cost) ==
+
+	Flechette (Fla-ch-ette) shells are a shotgun only ammo type, which contain 
```
I love the pronunciation guide. I was wondering this myself! :joy:

Turtlelord26 replied:
ch is not its own syllable. 
via Merriam-Webster: fle-ˈshet

Reveraine replied:
I think, with Turtle's suggestion, this will be good to go.

bleehu commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -601,6 +601,23 @@ biologic targets and armor.
+They are quite effective at rendering flesh and circuitry to a greater weakened state,
+and can pierce through materials to ensure the target is reached. These particular type
+of pointed bits are great at travelling greater distances, even while inside a target,
+having greater chance to hit something vital, causing great pain/malfunctions. 
```
I love the inclusion of cybernetic targets in the flavor text! Great job!

bleehu commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -601,6 +601,23 @@ biologic targets and armor.
+the save, target is inflicted with severe pain/ circuit damage and takes a -1 to all
+accuracy rolls. This lasts untilwound is healed or they are dead. If the shot fails 
+to pierce armor but hits it, reduce damage dealt by 50%, then reduce further by
+(5* (the difference in APL+1)).
```
blocking: I like this effect, but it asks the GM to do a lot of math to figure out what the armor damage is on-the-fly. I'm concerned that it's going to slow the game down.

Reveraine replied:
I agree with Bleehu, but I think it can be simplified by making it into a status effect, and maybe not have the armor affected part(or have that just be explained in its AP level.

Non-blocking. There are a few places (lines 616, and 618) where there are spaces missing between words.

bleehu commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -601,6 +601,23 @@ biologic targets and armor.
+having greater chance to hit something vital, causing great pain/malfunctions. 
+
+When using Flechette shells, increase damage 40% when shots pierce or miss armor. APL 
+of all guns ammo is used in is increased by 1. All range brackets gain a 5m increase,
```
nonblocking: Consider rephrasing "all guns ammo is used in is increased by." It works, but I think we can make it a little easier to read.

Reveraine replied:
I agree. As a suggestion, perhaps "When in use, the APL of weapon is increased by 1" or "+1 to APL of any weapon loaded with these shell."
This will be influenced by a discussion I started today on standardizing modification language.

bleehu reviewed (changes requested):
Requesting some minor changes, specifically how much damage the ammo does to armor. Other than that, you're off to a great start! 
The level 10 requirement and steep price I think justifies how nasty this ammo is.

Turtlelord26 reviewed (changes requested):
I like a lot of what I am seeing here conceptually, but I'm concerned that the ammunition is doing too many things, and that all of them are positive. There's almost no tactical puzzle of when to use these over normal buckshot - they're just better. My cursory research indicates we could consider an accuracy drop as one possible drawback, but there're probably many different ways we could take this.

I also wonder how necessary the shock save is, particularly after we've decided we don't very much like the universal 70 damage shock save. I am also concerned that the result of failing your save is an enemy debuff, which is a new departure in ammunition effects and possibly lets classes like soldier and specialist encroach on territory we want to reserve to the more supporty classes. Design: discuss. @bleehu 

Finally, and this is a pretty nitpicky bit, to help with classifying this a future shotgun ammo development I would like "shotgun-only ammo type" to instead read "shotgun-only spreading shot". I'm intending to have formal shotgun ammo categories of spreading shot vs solid shot, so that we can keep the weapon accuracy-range tables as simple as possible.

bleehu commented:
It's important for us to reference Issue #496  so that we know to close the issue when this ships.

Reveraine commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -601,6 +601,23 @@ biologic targets and armor.
+== Flechette Shells ($2.5 x mag cost) ==
+
+	Flechette (Fla-ch-ette) shells are a shotgun only ammo type, which contain 
+many flight sustaining, small, deadly, sharp pointed bits of a titanium alloy. 
```
Non-blocking: We can probably condense this sentence to contain less adjectives and read smoother. 
Such as: contain titanium alloy shrapnel (which I think is a common enough noun that most players will understand, and if we're worried about it, we can put it in the glossary)

Reveraine commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -601,6 +601,23 @@ biologic targets and armor.
+many flight sustaining, small, deadly, sharp pointed bits of a titanium alloy. 
+They are quite effective at rendering flesh and circuitry to a greater weakened state,
+and can pierce through materials to ensure the target is reached. These particular type
+of pointed bits are great at travelling greater distances, even while inside a target,
```
Non-blocking: Suggest changing pointed bits to shrapnel.

Reveraine commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -601,6 +601,23 @@ biologic targets and armor.
+
+When using Flechette shells, increase damage 40% when shots pierce or miss armor. APL 
+of all guns ammo is used in is increased by 1. All range brackets gain a 5m increase,
+except R1, which gains a 10m increase. Upon doing damage greater than 80, the target
```
This can probably be reworded to be shortened. Such as: "Gain +10m to R1, and +5m to all other range brackets."

Reveraine reviewed (changes requested):
There are just a few small things that should be changed before it's merged. Some spelling errors and such that are better caught and fixed now than later.

Masterdiekillsu committed:
Updated: Flechette Rounds

A number of changes have been made.
-Word use and language has been changed to reflect ease of reading.
-Weapon pronunciation has been altered.
-Range brackets have been given a debuff to encourage close range fighting
-The additional armor hit penalty has been removed for ease of number crunching and to help balance the weapon.

Masterdiekillsu commented:
Changes have been made According to your suggestions!

Turtlelord26 commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -603,21 +603,15 @@ target, it deals an extra 45 damage.
-to pierce armor but hits it, reduce damage dealt by 50%, then reduce further by
-(5* (the difference in APL+1)).
+	Flechette (Fle - shet) shells are a shotgun only spreading shot, which contains 
+flight sustaining, titanium alloy shrapnel. They are quite effective at rendering flesh
```
I'm sorry, I really should have caught this sooner. Flechettes are not well-described as shrapnel, as they are not fragments, randomly shaped, or propelled by an explosion.
> "fragments of a bomb, shell, or other object thrown out by an explosion."

Would prefer instead "darts" or an equivalent.

Turtlelord26 commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -603,21 +603,15 @@ target, it deals an extra 45 damage.
-(5* (the difference in APL+1)).
+	Flechette (Fle - shet) shells are a shotgun only spreading shot, which contains 
+flight sustaining, titanium alloy shrapnel. They are quite effective at rendering flesh
+and circuitry to a greater weakened state, and can pierce through materials to ensure 
```
Nitpick: "greater weakened state" does not flow well. Perhaps "effective at [damaging/weakening] flesh and circuitry".
Nitpick: "to ensure the target is reached" is passive voice. Sometimes passive voice is helpful, but I don't think it helps here. Consider replacing with "to reach the target" or just stopping the sentence at 'materials'.

Turtlelord26 commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -603,21 +603,15 @@ target, it deals an extra 45 damage.
+flight sustaining, titanium alloy shrapnel. They are quite effective at rendering flesh
+and circuitry to a greater weakened state, and can pierce through materials to ensure 
+the target is reached. These particular type of pointed bits are great at travelling short 
+distances, but looses trajectory the greater the distance. 
```
sp: "lose"

Turtlelord26 commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -603,21 +603,15 @@ target, it deals an extra 45 damage.
+distances, but looses trajectory the greater the distance. 
+
+When using Flechette shells, increase damage 40% when shots pierce or miss armor. 
+When in use, the APL of weapon is increased by 1. Gain +10m to R1, -5m to R2, and a -10m to R3. 
```
From a mechanical perspective, I need to know how the priority works with these range modifiers.
Specifically, if i have a weapon with R1 10m and R2 20m, then do R1 and R2 become 15 or 20?

Turtlelord26 commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -603,21 +603,15 @@ target, it deals an extra 45 damage.
+
+When using Flechette shells, increase damage 40% when shots pierce or miss armor. 
+When in use, the APL of weapon is increased by 1. Gain +10m to R1, -5m to R2, and a -10m to R3. 
+If the shot fails to pierce armor but hits it, reduce damage dealt by 50%.
```
This could be more clearly phrased as "Flechettes deal 50% damage to armor."

Turtlelord26 reviewed (changes requested):
I'm sorry to hit you with so many nitpicks, especially on the second round of reviews, but we're getting there.

I particularly like the use of both positive and negative range increment modifiers at the same time, with good flavor justification, but since that's new it's got a mechanical issue we haven't had to solve yet (see inline note).

Masterdiekillsu committed:
Updated nitpicks

Mostly focused in the second paragraph. Some extra details were added, and those will probably need to be reviewed for flow and how it fits with the item description. We're almost there

bleehu commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -601,6 +601,21 @@ biologic targets and armor.
+flight sustaining, titanium alloy darts. They are quite effective at weakening flesh
+and circuitry, and can pierce through materials to reach its target. These particular 
+type of pointed bits are great at travelling short distances, but lose trajectory 
+the greater the distance. 
```
good description! :)

bleehu commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -601,6 +601,21 @@ biologic targets and armor.
+type of pointed bits are great at travelling short distances, but lose trajectory 
+the greater the distance. 
+
+When using Flechette shells, increase damage 40% when shots pierce or miss armor. 
```
need to indent this line; non-blocking.

bleehu commented on Items/04_00_WeaponAttachments.txt:
```diff
@@ -601,6 +601,21 @@ biologic targets and armor.
+holds priority over the other brakets. For example, if R1 becomes 20m, and R2 becomes 15m, then 
+R1 is what will be used, and R2 will be disrarded entirely. If R3 does this as well, then R1 or 
+R2 will be used in its place, depending on which range R3 overlaps with. If any of the range 
+brackets are the same, the lowest bracket will be used. Ex: R1 instead of R2/R3, R2 instead of R3.
```
The discussion about which range brackets take priority over which is a little wordy, but I think it's able to be played and tested. I say shipit!

Turtlelord26 replied:
Sp. Disregarded

bleehu reviewed (approved):
shipit!
